Assets/Scripts/Agent/Zombie/ZombieStateMachine.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Com.StudioTBD.CoronaIO.FMS;
+using UnityEngine.AI;
+using Com.StudioTBD.CoronaIO.Agent.Human;
+
+namespace Com.StudioTBD.CoronaIO.Agent.Zombie
+{
+    public class ZombieStateMachine : StateMachine
+    {
+        private DataHolder _dataHolder;
```

Comment:
Make your own "data holder". Each agent should have it's own data holder since this is how the States share data.